src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -648,10 +699,28 @@ private[fastq] object FastqDemultiplexer {
     */
 
     def keep(omitFailingReads: Boolean, omitControlReads: Boolean): Boolean = {
-      val keepByQc      = if (!omitFailingReads) true else passQc
+      val keepByQc = if (!omitFailingReads) true else passQc
       val keepByControl = if (!omitControlReads) true else !isControl
       keepByQc && keepByControl
     }
+
+    /** A function to mask bases that have a quality score less than or equal to a specified threshold
```

Comment:
Double check me, but I think it is less than only.
```suggestion
    /** A function to mask bases that have a quality score less to a specified threshold
```